tests/test_ir.py

```diff
@@ -30,6 +30,9 @@ def test_ops_accessor():
 
     assert d.results[0] != c.results[0]
 
+    assert c.lhs.definition is a and c.rhs.definition is b
```

Comment:
Why the and here? Wouldn't two asserts make more sense and give better debug info when something breaks?

Reply:
No rational reason, just felt natural to write. I like that about Python! Will switch to two asserts

Reply:
Fair enough, sometimes python pretty much translates to english :sweat_smile: